### DISCUSSION TITLE
Unmanaged certificates - how to get them managed?

### DISCUSSION THREAD

**Discussion by dazinator** [polymind-inc/acmebot#439] [Q&A] [answered]
I have just got this up and running, pointing to an existing keyvault that has a couple of lets encrypt certificates already present.

The dashboard shows them under "Unmanaged" at the moment:

![image](https://user-images.githubusercontent.com/3176632/148562386-f7e1c332-d764-4ffe-847d-d5d06d75d359.png)

I'd like to get them into a "managed" state, so that they will be auto renewed. Any guidance on the correct path to achieve this?

**Comment by shibayan** [comment, marked as answer]
An easy way to do this is to open Details and run Renew. A new certificate will be created under Acmebot management via ACME according to the current certificate settings.

**Reply by dazinator** [reply]
worked like a treat, thanks
